src/TestSuite/Fixture/Extension/PHPUnitStartedSubscriber.php

```diff
@@ -34,8 +35,8 @@ public function notify(Started $event): void
         $helper = new ConnectionHelper();
         $helper->addTestAliases();
 
-        $enableLogging = in_array('--debug', $_SERVER['argv'] ?? [], true);
-        if ($enableLogging) {
+        $enableLogging = env('LOG_QUERIES') ?? 0;
```

Comment:
Should we be using CAKE_ prefixes for our official env variables? I know we have DEBUG, but users can change that.

Reply:
I was torn on this. I think having the CAKE_ prefix is helpful for runtime configuration as you could have lots of environment variables. But this would only be used in local development, so I didn't think the prefix was necessary.

Reply:
If the name is clear, I think it's ok. LOG_QUERIES seems clear. I just know it's not obvious what it controls when read out of context.